Initial setup

danieljy opened the pull request:
Take a look!

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/sidewalklabs/opentripplanner/1)
<!-- Reviewable:end -->

m12i commented:
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

---

Reviewed 19 of 19 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved.

---

*[Dockerfile, line 11 at r1](https://reviewable.io/reviews/sidewalklabs/opentripplanner/1#-LAxxbQq4g68PbRGEOQR:-LAxxbQq4g68PbRGEOQS:b6n3jsg) ([raw file](https://github.com/sidewalklabs/opentripplanner/blob/7fd5b4a92b83e953bae3bcdd8520965694023164/Dockerfile#L11)):*
> ```
> # TODO(danieljy): Load a much larger road network than just downtown DC.
> ```

is TODO still needed?

---

*[coord/README.md, line 7 at r1](https://reviewable.io/reviews/sidewalklabs/opentripplanner/1#-LAxzsGdBbfN17sCYwb4:-LAxzsGdBbfN17sCYwb5:b1qcf4t) ([raw file](https://github.com/sidewalklabs/opentripplanner/blob/7fd5b4a92b83e953bae3bcdd8520965694023164/coord/README.md#L7)):*
> ```
> run-helpers.sh`
> ```

running this command while in `coord` directory fails:

```bash
./run-helpers.sh: line 5: ./coord/run-test-gbfs.sh: No such file or directory
./run-helpers.sh: line 6: ./coord/run-frontend.sh: No such file or directory
./run-helpers.sh: line 7: kill: (2113) - No such process
```

it should be `./coord/run-helper.sh` to work

---

*[coord/README.md, line 9 at r1](https://reviewable.io/reviews/sidewalklabs/opentripplanner/1#-LAy0cCb0-bP_VeGR9zB:-LAy0cCb0-bP_VeGR9zC:boueqce) ([raw file](https://github.com/sidewalklabs/opentripplanner/blob/7fd5b4a92b83e953bae3bcdd8520965694023164/coord/README.md#L9)):*
> ```Markdown
> `run-helpers.sh`
> 
> Run the actual OTP server in another (avoid restarting, this takes time!)
> ```


`avoid restarting` seems vague,  Can we mention when do we have to restart the OTP server. I mean which kind of changes?

---

*[coord/README.md, line 11 at r1](https://reviewable.io/reviews/sidewalklabs/opentripplanner/1#-LAy-N1Q17bhPZ02Zg5w:-LAy-N1Q17bhPZ02Zg5x:b-ogfp6k) ([raw file](https://github.com/sidewalklabs/opentripplanner/blob/7fd5b4a92b83e953bae3bcdd8520965694023164/coord/README.md#L11)):*
> ```
> run-otp.sh`
> ```

` ./coord/run-otp.sh`

---

*[coord/README.md, line 11 at r1](https://reviewable.io/reviews/sidewalklabs/opentripplanner/1#-LAy-dW71ZD5QeX3NzNZ:-LAy-dW71ZD5QeX3NzN_:b-hjbpbv) ([raw file](https://github.com/sidewalklabs/opentripplanner/blob/7fd5b4a92b83e953bae3bcdd8520965694023164/coord/README.md#L11)):*
> ```Markdown
> Run the actual OTP server in another (avoid restarting, this takes time!)
> 
> `run-otp.sh`
> ```

Also add:

point to `http://localhost:10000` for GBFS and `http://localhost:9000` for OTP GUI

---

*[coord/README.md, line 11 at r1](https://reviewable.io/reviews/sidewalklabs/opentripplanner/1#-LAy05UdEnRkeVd5qq67:-LAy05UdEnRkeVd5qq68:b3bopg6) ([raw file](https://github.com/sidewalklabs/opentripplanner/blob/7fd5b4a92b83e953bae3bcdd8520965694023164/coord/README.md#L11)):*
> ```Markdown
> Run the actual OTP server in another (avoid restarting, this takes time!)
> 
> `run-otp.sh`
> ```

When `./coord/run-otp.sh` it takes a while for the program to print out information. Would be nice to print a line saying `Server is loading, it may take a while`.

---

*[src/client/js/otp/config.js, line 50 at r1](https://reviewable.io/reviews/sidewalklabs/opentripplanner/1#-LAy3sPz28wddAZjBGi2:-LAy3sQ-B7W5NJVyevkk:b7qs50n) ([raw file](https://github.com/sidewalklabs/opentripplanner/blob/7fd5b4a92b83e953bae3bcdd8520965694023164/src/client/js/otp/config.js#L50)):*
> ```
> 'http://localhost:9000'
> ```


'http://localhost:9000' seems the same UI as 'http://localhost:8080', what's the difference?

---


*Comments from [Reviewable](https://reviewable.io/reviews/sidewalklabs/opentripplanner/1#-:-LAy4IfaEPlTJbSVUzYV:bnfp4nl)*
<!-- Sent from Reviewable.io -->

danieljy commented:
@m12i I've addressed all your concerns.

I'm merging in.

---

Review status: 17 of 19 files reviewed at latest revision, 7 unresolved discussions, all commit checks successful.

---

*[Dockerfile, line 11 at r1](https://reviewable.io/reviews/sidewalklabs/opentripplanner/1#-LAxxbQq4g68PbRGEOQR:-LAy5G3X61NA1WGWbBi8:brxkrfo) ([raw file](https://github.com/sidewalklabs/opentripplanner/blob/7fd5b4a92b83e953bae3bcdd8520965694023164/Dockerfile#L11)):*
<details><summary><i>Previously, m12i (Mahmood Rahmani) wrote…</i></summary><blockquote>

> ```
> # TODO(danieljy): Load a much larger road network than just downtown DC.
> ```

is TODO still needed?
</blockquote></details>

@noahgreenbaum Can you let us know if this area covers the union of all DC bike share areas?

http://download.geofabrik.de/north-america/us/district-of-columbia.html

---

*[coord/README.md, line 7 at r1](https://reviewable.io/reviews/sidewalklabs/opentripplanner/1#-LAxzsGdBbfN17sCYwb4:-LAy5jTM0a-ekokGUAip:b-896fix) ([raw file](https://github.com/sidewalklabs/opentripplanner/blob/7fd5b4a92b83e953bae3bcdd8520965694023164/coord/README.md#L7)):*
<details><summary><i>Previously, m12i (Mahmood Rahmani) wrote…</i></summary><blockquote>

> ```
> run-helpers.sh`
> ```

running this command while in `coord` directory fails:

```bash
./run-helpers.sh: line 5: ./coord/run-test-gbfs.sh: No such file or directory
./run-helpers.sh: line 6: ./coord/run-frontend.sh: No such file or directory
./run-helpers.sh: line 7: kill: (2113) - No such process
```

it should be `./coord/run-helper.sh` to work
</blockquote></details>

Done.

---

*[coord/README.md, line 9 at r1](https://reviewable.io/reviews/sidewalklabs/opentripplanner/1#-LAy0cCb0-bP_VeGR9zB:-LAy63UuExVRHdUOuZO2:b-896fix) ([raw file](https://github.com/sidewalklabs/opentripplanner/blob/7fd5b4a92b83e953bae3bcdd8520965694023164/coord/README.md#L9)):*
<details><summary><i>Previously, m12i (Mahmood Rahmani) wrote…</i></summary><blockquote>

`avoid restarting` seems vague,  Can we mention when do we have to restart the OTP server. I mean which kind of changes?
</blockquote></details>

Done.

---

*[coord/README.md, line 11 at r1](https://reviewable.io/reviews/sidewalklabs/opentripplanner/1#-LAy-N1Q17bhPZ02Zg5w:-LAy63wn-BPp1Q0CCb4p:b-896fix) ([raw file](https://github.com/sidewalklabs/opentripplanner/blob/7fd5b4a92b83e953bae3bcdd8520965694023164/coord/README.md#L11)):*
<details><summary><i>Previously, m12i (Mahmood Rahmani) wrote…</i></summary><blockquote>

> ```
> run-otp.sh`
> ```

` ./coord/run-otp.sh`
</blockquote></details>

Done.

---

*[coord/README.md, line 11 at r1](https://reviewable.io/reviews/sidewalklabs/opentripplanner/1#-LAy05UdEnRkeVd5qq67:-LAy7SeG17y3Dcoj_0XC:bnwwym6) ([raw file](https://github.com/sidewalklabs/opentripplanner/blob/7fd5b4a92b83e953bae3bcdd8520965694023164/coord/README.md#L11)):*
<details><summary><i>Previously, m12i (Mahmood Rahmani) wrote…</i></summary><blockquote>

When `./coord/run-otp.sh` it takes a while for the program to print out information. Would be nice to print a line saying `Server is loading, it may take a while`.
</blockquote></details>

Fixed this. Also added -it which lets you CTRL-C to stop the docker container.  

---

*[src/client/js/otp/config.js, line 50 at r1](https://reviewable.io/reviews/sidewalklabs/opentripplanner/1#-LAy3sPz28wddAZjBGi2:-LAy7bU575rdJvK85VI4:b-c3hbi1) ([raw file](https://github.com/sidewalklabs/opentripplanner/blob/7fd5b4a92b83e953bae3bcdd8520965694023164/src/client/js/otp/config.js#L50)):*
<details><summary><i>Previously, m12i (Mahmood Rahmani) wrote…</i></summary><blockquote>

> ```
> 'http://localhost:9000'
> ```


'http://localhost:9000' seems the same UI as 'http://localhost:8080', what's the difference?

</blockquote></details>

It's confusing because localhost:8080 is the actual backend, but they server their frozen version of the frontend from it too.

We run our own frontend on :9000, and this just configures our frontend to talk to the correct backend (8080) instead of defaulting to 9000.

Does that make sense?

---


*Comments from [Reviewable](https://reviewable.io/reviews/sidewalklabs/opentripplanner/1#-:-LAy8BTk6wCfeZs08CsJ:b-xj6hau)*
<!-- Sent from Reviewable.io -->
